manual/_posts/2021-04-22-wet-lab-tips-tricks.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Wet lab tips and tricks"
+description: "Miscellaneous tips and tricks for the wet lab in Waddington building"
+about: true
+author_handle: ew
+tags: [labmanual, wetlab]
+---
+{% include JB/setup %}
+
+# Wet lab tips and tricks
+
+This is a page for the random tips and tricks
+
+- how to operate machines
+- how to move data between machines
+- how to get stuff, e.g. dry ice
+
+Remember, this is going to be on a public website, so no passwords or personal info.
+
+## Dry ice
+
+Mostly for dry ice we get this from the Swann building lobby; ensure you have card access to the Swann building.
+Before getting dry ice, please check with Greg Anderson that there is some available – it is purchased by the Wellcome centre and they don’t always have enough to share.
+
+HELP: HOW DO WE ORDER DRY ICE IF THEY DON'T?
+
+## DeNovix / nanodrop
+
+To get data from the nanodrop to datastore / lab notebook, ...
+HELP
+
+## Lab tablet computers
+
+HELP
```

Comment:
## Ordering outside of Sciquest

If you need to order something that is not available on Sciquest and asks for a lab address, please C/O Graham Nicolson if possible and use the address:

University of Edinburgh
School of Biological Sciences
G27 Roger Land Building
Alexander Crum Brown Road
Edinburgh
EH9 3FF

This ensures the item is delivered to Stores and which is helpful for deliveries outside of core hours.

Reply:
I added this in to the document.